src/integration/java/io/pinecone/helpers/TestIndexResourcesManager.java

```diff
@@ -53,6 +58,14 @@ public static TestIndexResourcesManager getInstance() {
         return instance;
     }
 
+    public  Index getServerlessIndexConnection() {
+        return this.getInstance().pineconeClient.getIndexConnection(this.serverlessIndexName);
+    }
+
+    public AsyncIndex getServerlessAsyncIndexConnection() {
+        return this.getInstance().pineconeClient.getAsyncIndexConnection(this.serverlessIndexName);
+    }
```

Comment:
nit, you don't need to specify `this.` in most of these methods. The only time you need to specify `this.` is when there is a conflicting parameter, like there would typically be in a setter method.

```java
public void setName(String name) {
    this.name = name;
}
```

Reply:
ah ty!